vit-model-training/walkthrough.ipynb

```diff
@@ -2,24 +2,20 @@
  "cells": [
```

Comment:
<p>Can you move the Paperspace stuff above the dataset description?  And in the dataset description add a not that it will be preloaded?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/Graphcore-Pytorch-fork/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>